### PR TITLE
DOC: Add CoC to the README

### PR DESCRIPTION
**Pull request by sparalic**
- [ ] closes # xxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,3 +233,6 @@ You can also triage issues which may include reproducing bug reports, or asking
 Or maybe through using pandas you have an idea of your own or are looking for something in the documentation and thinking ‘this can be improved’...you can do something about it!
 
 Feel free to ask questions on the [mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata) or on [Gitter](https://gitter.im/pydata/pandas).
+
+## Contributor Code of Conduct
+Pandas has a strict code of conduct that all contributors and maintainers of this project must follow. More information can be found at: [Pandas Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```
Comment (datapythonista) [inline README.md]: Thanks for this @sparalic.

I think it makes more sense to have this as a paragraph of the `Contributing to pandas` section, not a section of its own.

Also, we don't capitalize the name `pandas`. Other than that, lgtm.

Comment (sparalic) [reply README.md]: @datapythonista thank you! I made the changes you suggested above

File: README.md [hunk 2]
```diff
@@ -233,3 +233,5 @@ You can also triage issues which may include reproducing bug reports, or asking
 Or maybe through using pandas you have an idea of your own or are looking for something in the documentation and thinking ‘this can be improved’...you can do something about it!
 
 Feel free to ask questions on the [mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata) or on [Gitter](https://gitter.im/pydata/pandas).
+
+As contributors and maintainers to this project you are expected to abibe by pandas strict code of conduct. More information can be found at: [Pandas Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```
Comment (datapythonista) [inline README.md]:
```suggestion
As contributors and maintainers to this project you are expected to abibe by pandas strict code of conduct. More information can be found at: [Contributor Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```

File: README.md [hunk 3]
```diff
@@ -233,3 +233,5 @@ You can also triage issues which may include reproducing bug reports, or asking
 Or maybe through using pandas you have an idea of your own or are looking for something in the documentation and thinking ‘this can be improved’...you can do something about it!
 
 Feel free to ask questions on the [mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata) or on [Gitter](https://gitter.im/pydata/pandas).
+
+As contributors and maintainers to this project you are expected to abibe by pandas strict code of conduct. More information can be found at: [Contributor Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```
Comment (datapythonista) [inline README.md]:
```suggestion
As contributors and maintainers to this project you are expected to abide by pandas strict code of conduct. More information can be found at: [Contributor Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```
Sorry didn't see that before, but this looks like a typo, isn't it?

Comment (sparalic) [reply README.md]: @datapythonista good catch, yes it is.

**Review by datapythonista — APPROVED** [review]
lgtm

File: README.md [hunk 4]
```diff
@@ -233,3 +233,5 @@ You can also triage issues which may include reproducing bug reports, or asking
 Or maybe through using pandas you have an idea of your own or are looking for something in the documentation and thinking ‘this can be improved’...you can do something about it!
 
 Feel free to ask questions on the [mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata) or on [Gitter](https://gitter.im/pydata/pandas).
+
+As contributors and maintainers to this project you are expected to abide by pandas strict code of conduct. More information can be found at: [Contributor Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```
Comment (gfyoung) [inline README.md]:
```suggestion
As contributors and maintainers to this project you are expected to abide by pandas' strict code of conduct. More information can be found at: [Contributor Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```

Comment (gfyoung) [reply README.md]: Note the apostrophe after "pandas" in "pandas strict code of conduct"

Comment (sparalic) [reply README.md]: @gfyoung great catch!

File: README.md [hunk 5]
```diff
@@ -233,3 +233,5 @@ You can also triage issues which may include reproducing bug reports, or asking
 Or maybe through using pandas you have an idea of your own or are looking for something in the documentation and thinking ‘this can be improved’...you can do something about it!
 
 Feel free to ask questions on the [mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata) or on [Gitter](https://gitter.im/pydata/pandas).
+
+As contributors and maintainers to this project you are expected to abide by pandas' strict code of conduct. More information can be found at: [Contributor Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```
Comment (gfyoung) [inline README.md]:
```suggestion
As contributors and maintainers to this project, you are expected to abide by pandas' strict code of conduct. More information can be found at: [Contributor Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```

Comment (gfyoung) [reply README.md]: Sorry, I should have caught that one as well.

We try our best to follow by an additional code of conduct that is proper, grammatical English 😁

Comment (sparalic) [reply README.md]: @gfyoung  thank you! I totally understand

**Comment by TomAugspurger** [comment]
@sparalic can you merge master & repush? Looks like there was an unrelated CI failure.

File: README.md (hunk 5 above)
Comment (TomAugspurger) [inline README.md]: Do we need "strict" here? I'd like it a bit more without that.

Comment (sparalic) [reply README.md]: @TomAugspurger I made the changes you suggest above. Thanks!

**Review by datapythonista — CHANGES_REQUESTED** [review]
@sparalic looks like some files were accidentally added to this PR. If you are using `git commit -am`, it'd probably be safer to not do it, and add files explicitly. Also, after pushing (or before with `git diff upstream/master`), it's good to make sure that the PR contains the changes you expect.

If you can please remove the files, I think after that we're ready to merge. Thanks!

**Comment by sparalic** [comment]
@datapythonista I explicitly pushed the file I updated, and did a `git diff upstream/master` to confirm the changes. Is that wrong?

**Comment by datapythonista** [comment]
Not sure what was the problem. You can have a look at the [Files changed](https://github.com/pandas-dev/pandas/pull/27780/files) tab, and you'll see that there are 4 files that are unrelated to what you're changing, and that we don't want to merge into the repo.

Also, I think you reverted some of the changes you already made from a review. We will check later what went wrong, for now you can delete the files in your branch, and leave exactly what you want to get merged in pandas, and then push the branch again.

**Comment by gfyoung** [comment]
@sparalic : FYI, if you're having issues pushing, we can definitely help you out instead of your closing the PR.

**Pull request closed without merging**